tests/infra/member.py

```diff
@@ -81,7 +79,12 @@ def set_active(self):
         # Use this with caution (i.e. only when the network is opening)
         self.status_code = MemberStatus.ACTIVE
 
-    def propose(self, remote_node, proposal, has_proposer_voted_for=True):
+    def propose(
+        self,
+        remote_node,
+        proposal,
+        has_proposer_voted_for=True,
```

Comment:
Unrelated nitpick: Its a little unfortunate that _Black_ does this. If it ever inserts a trailing comma, it won't remove it. There's several issues around this on their repo. If I was writing an opinionated, unconfigurable formatter with an aim to "produce the smallest diffs possible", I'd enforce that this trailing-comma was _always_ present, and lists of all kinds were always split over multiple lines. Remove the spill/unspill hysteresis entirely. Rant over.